packages/ceu-lang/build.sh

```diff
@@ -0,0 +1,22 @@
+TERMUX_PKG_HOMEPAGE="https://github.com/ceu-lang/ceu"
+TERMUX_PKG_DESCRIPTION="The Structured Synchronous Reactive Programming Language Céu"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_MAINTAINER="@termux"
+TERMUX_PKG_SRCURL="https://github.com/ceu-lang/ceu.git"
+TERMUX_PKG_GIT_BRANCH="master"
+_COMMIT="5e0c8d3004ad98658ffe82823ad8303a8d371064"
+TERMUX_PKG_VERSION="2019.07.17"
+TERMUX_PKG_BUILD_IN_SRC=true
+TERMUX_PKG_BUILD_DEPENDS="lua53, lua-lpeg, liblua53"
```

Comment:
Is this based on test builds on device? For cross-compiling I would expect lua53 to be required in docker image ([setup-ubuntu.sh](https://github.com/termux/termux-packages/blob/master/scripts/setup-ubuntu.sh)), and liblua53 to be a runtime dependency, in `TERMUX_PKG_DEPENDS`

Reply:
The ceu compiler  is actually a `lua5.3` script. I tested ceu on target (aarch64) built with this: 
```
TERMUX_PKG_BUILD_DEPENDS="lua53, lua-lpeg"
TERMUX_PKG_DEPENDS="lua53, lua-lpeg, liblua53"
```